### ISSUE TITLE
Feature-Request: Avoid connection lines to cross over artboards

### ISSUE THREAD

**Issue by Adam-CS** [abynim/UserFlows.sketchplugin#24]
Hi and congrats! 
Awesome plugin!
I would completely fall in love with this plugin if the generated flow boards didn't look like spaghetti monster porn. 
It would be great if there was a way to keep the connection lines from crossing over the artboards but instead move in the space in between the artboards either automatically or manually by hand.
I understand this isn't at all easy but I think it would make a world of difference in the generated output and the flexibility of use!
Cheers!

**Comment by itsbrennan** [comment]
Agreed, if we could have these vectored lines be defaulted to "Straight" (or have the option) instead of "Disconnected" that would really clean up these artboards by having the lines break at 90°! I've been changing them manually. Love the plugin :)

**Comment by erutan** [comment]
Agreed as well.

**Comment by JMESO** [comment]
@Bmart2913 - how have you been changing them manually? I don't seem to be able to select the individual connectors
